### PR TITLE
feat: read/write labelplacement

### PR DESCRIPTION
**Pull request by jansule**
This adds support for label placements `point` and `line`.

### CODE REVIEW

File: src/SldStyleParser.ts [hunk 1]
```diff
@@ -695,18 +701,28 @@ export class SldStyleParser implements StyleParser<string> {
...
+        const displacement = get(placement, 'PointPlacement.Displacement');
+        if (!isNil(displacement)) {
+          const x = get(displacement, 'DisplacementX.#text');
+          const y = get(displacement, 'DisplacementY.#text');
+          textSymbolizer.offset = [
+            Number.isFinite(x) ? numberExpression(x) : 0,
+            Number.isFinite(y) ? numberExpression(y) : 0,
+          ];
+        }
+        const rotation = get(placement, 'PointPlacement.Rotation.#text');
```
Comment (KaiVolland) [inline src/SldStyleParser.ts:719]: Shouldn't this be:
```suggestion
        const rotation = get(pointPlacement, 'Rotation.#text');
```

**Review by KaiVolland — APPROVED** [review]
Just a minor note. But if the tests are ok with it I'm too. :)

File: src/SldStyleParser.ts (hunk 1 above)
Comment (jansule) [reply src/SldStyleParser.ts]: solved in https://github.com/geostyler/geostyler-sld-parser/pull/916/commits/9c455a64287da0dbd7ebc628705d7b575f52236b

**Pull request merged**